app/clients/item_client.rb

```diff
@@ -0,0 +1,19 @@
+class ItemClient
+  def self.get_url(url, query = nil)
+    conn = Faraday.new(url: 'http://localhost:3000') do |faraday|
+      faraday.params[:query] = query unless query.nil?
+    end
+
+    response = conn.get(url)
+
+    data = JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
Since this is the same for both `ItemClient` and `MerchantClient` you might consider creating a `BaseClient` for this method that each of Item and Merchant client's inherit from. That way, you only need to setup and manage this in one place.